docs/api.md

```diff
@@ -16,6 +16,7 @@
     - [[app-version]](#app-version)
     - [[build-version]](#build-version)
     - [[electron-version]](#electron-version)
+    - [[wvvmp]](#wvvmp)
```

Comment:
Hey, one last thing: what about renaming the option simply `widevine` ? `wvvmp` is a pain to remember and type.

Reply:
Oof... that makes way more sense...